packages/postgres-database/src/simcore_postgres_database/migration/versions/22404057a50c_resource_tracker_service_runs_helpers_.py

```diff
@@ -0,0 +1,35 @@
+"""resource_tracker_service_runs helpers for missing heartbeat
+
+Revision ID: 22404057a50c
+Revises: 2a4b4167e088
+Create Date: 2023-10-25 19:17:29.928871+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "22404057a50c"
+down_revision = "2a4b4167e088"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "resource_tracker_service_runs",
+        sa.Column("service_run_status_msg", sa.String(), nullable=True),
```

Comment:
why is this message nullable? what is the idea here?

Reply:
This is a flexible column just for us to make some notes either automatically from code or manually if some intervention is needed. In theory default might be empty string and it does need to be nullable.